events/pull-meetup.js

```diff
@@ -37,22 +36,36 @@ function finish (events) {
 // This is nice when testing if you cache the response
 // finish(JSON.parse(require('fs').readFileSync('./meetup.json').toString()))
 
-function _go (u) {
-  request(u, (e, resp, body) => {
-    const results = body.results
-    results.forEach((result) => {
+function pull (opts) {
+  request(opts, (err, resp, body) => {
+    if (err || resp.statusCode !== 200) {
+      throw (err || new Error('response not 200' + resp.statusCode))
+    }
+
+    body.results.forEach((result) => {
       const title = result.name.toLowerCase()
-      if (title.indexOf('nodeschool') !== -1) return
-      if (title.indexOf('mongodb') !== -1 && title.indexOf('node') === -1) return
-      if (title.indexOf('find a tech job') !== -1 && title.indexOf('node') === -1) return
-      // if (title.indexOf('node') !== -1) return allresults.push(result)
-      allresults.push(result)
+
+      if (title.includes('nodeschool')) return
+      if (title.includes('mongodb') && title.includes('node')) return
+      if (title.includes('find a tech job') && title.includes('node')) return
+
+      results.push(result)
     })
+
     if (body.meta.next) {
-      _go(body.meta.next)
+      pull(Object.assign({}, defaults, { url: body.meta.next }))
```

Comment:
Any reason you didn't go for `Object.assign({ url: body.meta.next }, defaults)`?